user_manual/pim/sync_osx.rst

```diff
@@ -2,16 +2,51 @@
 Synchronizing with macOS
 ========================
 
-With Nextcloud 14+ and its CalDAV-backend, syncing your Calendars with your macOS-device is easy.
+Setup your Accounts
+-------------------
 
-To use Nextcloud with macOS Calendar you will need to use the following steps:
+Adding your Calendar and Contacts resources from Nextcloud Groupware to macOS
+native applications is easy. In the following steps you will add your server
+resources for **CalDAV** (Calendar) and **CardDAV** (Contacts) to your Nextcloud.
 
-1. Open the Calendar-app in your Nextcloud (https://cloud.example.com/apps/calendar)
-2. Navigate into the app-settings (lower left corner) and copy the iOS/macOS CalDAV address provided there
-3. Open the system preferences of your macOS-device
-4. Navigate to "Online-Accounts"
-5. Click on "Add another account..." and click on "CalDAV-Account"
-6. Select "Manually" as Account-Type and type in your username or email-address of your Nextcloud-instance as username, and your app-password/token or password as password.
-7. Paste the previous copied (step 2) iOS/macOS CalDAV address into the server address field and click on "Login".
+1. Open the **system preferences** of your macOS device
 
-You can now select, with which applications you want to use this CalDAV resource. In the most cases, this will be "Calendar", sometimes you may also want to use it for your Tasks and reminders.
+2. Navigate to **Online-Accounts**
+
+.. figure:: /images/macos_1.png
+
+3. Click on **Add Other Account...** and click on **CalDAV-Account** for Calendar
+   or **CardDAV-Account** for Contacts.
+
+.. figure:: /images/macos_2.png
+
+.. note:: You can not setup Calendar/Contacts together. You need to setup them
+          in **separate accounts**.
+
+4. Select **Advanced** as Account-Type and type in your respective credentials:
+
+   **Username**: Your Nextcloud username or email
+   **Passwort**: Your generated App-password/token
```

Comment:
"Password" :) Should there be a link to steps to create app passwords? People may not understand the why or how of app-specific passwords when they just come in to learn about calendars and contacts synchronisation.